src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14429,12 +14442,20 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                     goto Label13;
                 }
 
-                if (CapDiff > (Tolerance * Cap_Eva0)) NumIteCcap = 999;
+                if (CapDiff > (Tolerance * Cap_Eva0)) {
+                    NumIteCcap = 999;
+                    CyclingRatio = (TU_load + Pipe_Q) * C_cap_operation / Cap_Eva1;
+                } else {
+                    CyclingRatio = 1.0;
+                }
 
                 Ncomp = this->RatedCompPower * CurveValue(state, this->OUCoolingPWRFT(CounterCompSpdTemp), T_discharge, T_suction);
+                OUCondHeatRelease = Ncomp + Cap_Eva1;
```

Comment:
Condenser heat rejection = power + evap capacity. How does this calculation relate to lines 11706 and 11707? Is OUCondHeatRelease the same as Q_h_OU at line 11707? And where is Q_c_OU *= cyclingRatio? I am getting confused as to which variables account for cycling and which don't. Does `OUCondHeatRelease = Ncomp + Cap_Eva1;` account for cycling?

Reply:
Ncomp should probably account for cycling
OUCondHeatRelease was computed as the following in the `CounterCompSpdTemp > NumOfCompSpdInput case`
```
OUCondHeatRelease = Ncomp + CompEvaporatingCAPSpd(NumOfCompSpdInput);
```
So I basically followed this in the low-load approach. Originally I have the Ncomp and `OUCondHeatRelease` multiply by CyclingRatio (or maybe it should be RTF) in CalcCompH() function, but since the flow of the code is like 

```
        Ncomp = TU_CoolingLoad / this->CoolingCOP;
        Ncomp_new = Ncomp;
    Label10:;
        Q_h_OU = Q_c_TU_PL + Ncomp_new; // Ncomp_new may be updated during Iteration_Ncomp Label10

        ...

        // *VEF OU Compressor Simulation at cooling mode: Specify the compressor speed and power consumption
        this->VRFOU_CalcCompC(state,
                              TU_CoolingLoad,
                              Tsuction,
                              this->CondensingTemp,
                              Psuction,
                              T_comp_in,
                              h_comp_in,
                              h_IU_evap_in,
                              Pipe_Q_c,
                              CapMaxTc,
                              Q_h_OU,
                              CompSpdActual,
                              Ncomp,
                              CyclingRatio);

        if ((std::abs(Ncomp - Ncomp_new) > (Tolerance * Ncomp_new)) && (Counter < 30)) {
            Ncomp_new = Ncomp;
            Counter = Counter + 1;
            goto Label10;
        }
```
As the initial value of Ncomp (Ncomp = TU_CoolingLoad / this->CoolingCOP) does not have CyclingRatio in it, and the compressor is indeed going to cycling, then the cycling ratio will get multiplied multiple times, once each label10 loop. So I moved them to after the loop is completed.

```
OUCondHeatRelease = Ncomp + Cap_Eva1 = Ncomp + Cap_Eva0 / CyclingRatio;
```

```
OUCondHeatRelease * CyclingRatio = Ncomp * CyclingRatio + Cap_Eva0 = Ncomp * CyclingRatio + TU_load + Pipe_Q
```

Reply:
The plots you make of a transition from CyclingRatio = 1 to CyclingRatio < 1 (later in a day when the cooling load is decreasing) will show if these equations are correct. Maybe start there to determine how these equations need to be organized.

Reply:
The plot is in this comment https://github.com/NREL/EnergyPlus/pull/10353#discussion_r1521980123

Reply:
It doesn't seem like the cycling ratio is double-counted. If we want to make sure, maybe we could double-check the output result against the hand calculation result at a certain fixed timestep.